examples/dgfip_c/ml_primitif/main.ml

```diff
@@ -114,7 +114,7 @@ let run_test test_file =
 
   let annee_calc = M.annee_calc () in
 
-  let out = Printf.sprintf "output/%s.tgv" (Filename.basename test_file) in
+  let out = Printf.sprintf "%d.output/%s.tgv" annee_calc (Filename.basename test_file) in
```

Comment:
(feedback, no change needed) Technically, we can run a calculator compiled for year N on tests for year M where M <> N. I don't know if we want to test that though, in which case it's correct to specify `annee_calc here`. Note that the check below (line 123-124) remains necessary in case a test set for year N contains a test that incorrectly specify a year M <> N.